core/remote/change.js

```diff
@@ -20,12 +20,13 @@ export type RemoteFileUpdate = {sideName: 'remote', type: 'FileUpdate', doc: Met
 export type RemoteDirAddition = {sideName: 'remote', type: 'DirAddition', doc: Metadata, was: Metadata}
 export type RemoteDirDeletion = {sideName: 'remote', type: 'DirDeletion', doc: Metadata}
 export type RemoteDirDissociation = {sideName: 'remote', type: 'DirDissociation', doc: Metadata, was: Metadata}
-export type RemoteDirMove = {sideName: 'remote', type: 'DirMove', doc: Metadata, was: Metadata, needRefetch?: true}
+export type RemoteDirMove = {sideName: 'remote', type: 'DirMove', doc: Metadata, was: Metadata, needRefetch?: true, descendantDirMoves?: RemoteDescendantChange[] }
```

Comment:
Rename to `descendantMoves` since it can include files?
Or eventually to `childMoves` to be consistent with the existing Merge/Sync wording? (although descendant would definitely be more accurate in both cases)

Reply:
Are they move-only or can file moves include content changes too?

Reply:
> Are they move-only or can file moves include content changes too?

Answering myself: they can include content changes (hence the `previousMoveToSamePath()` change in the `RemoteWatcher`).

Reply:
renamed to `descendantMoves` in #1274 https://github.com/cozy-labs/cozy-desktop/pull/1274/commits/04a2fd8e0449c5179b4e3e17471887954863893b  

IMHO, being consistent with merge/sync would be more confusing.